runeq/resources/stream_metadata.py

```diff
@@ -1,26 +1,24 @@
 """
-V2 SDK functionality to support Stream Metadata operations.
+Fetch metadata about streams, including stream types.
 
 """
-
 import datetime
 from io import StringIO
-
-import pandas as pd
 from typing import Callable, Iterable, Iterator, List, Optional, Type, Union
 
+import pandas as pd
 
 from .client import GraphClient, StreamClient, global_graph_client
 from .common import ItemBase, ItemSet
 from .patient import Device, Patient
-from .stream import (
-    get_stream_availability, get_stream_csv, get_stream_data, get_stream_json
-)
+from .stream import get_stream_availability, get_stream_data
+from runeq.errors import RuneError
```

Comment:
nitpick: alphabetical order

Reply:
tbh, wasn't entirely sure about how to sort relative (e.g. `from .stream ...`) vs absolute (e.g. `from runeq.errors import...`) imports... by first character, I'd argue this is alphabetical?

Reply:
Gotcha, I'll keep that in mind too and follow the same convention!